type-definitions/Immutable.d.ts

```diff
@@ -1429,6 +1429,27 @@ declare module Immutable {
      */
     readonly size: number;
 
+    /**
+     * Returns a new OrderedMap also containing the new key, value pair. If an equivalent
+     * key already exists in this OrderedMap, it will be replaced while maintaining the
+     * existing order.
+     *
+     * <!-- runkit:activate -->
+     * ```js
+     * const { OrderedMapMap } = require('immutable')
+     * const originalMap = Immutable.OrderedMap({a:1, b:1, c:1})
```

Comment:
In the future - ensure lines like these are correct - this both imports a value that doesn't exist, and then proceeds to use a pattern we don't support in docs. I just added a commit to fix these issues.